tests/unit/common/config/test_service_config.py

```
@@ -282,3 +282,56 @@ def test_explicit_ui_type_overrides_non_tty(self, monkeypatch):
+        assert config.api_enabled is True
+
+    def test_api_enabled_true_from_env(self, monkeypatch) -> None:
+        monkeypatch.setattr(
+            "aiperf.common.environment.Environment.API_SERVER",
+            type("_FakeAPI", (), {"PORT": 9090})(),
+        )
+        config = ServiceConfig()
+        assert config.api_enabled is True
+
+    def test_api_port_and_host_set(self) -> None:
+        config = ServiceConfig(api_port=8080, api_host="0.0.0.0")
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)